frontend/src/metabase/querying/components/FilterPicker/FilterPickerBody/FilterPickerBody.tsx

```diff
@@ -60,7 +60,7 @@ function getFilterWidget(column: Lib.ColumnMetadata) {
   if (Lib.isNumeric(column)) {
     return NumberFilterPicker;
   }
-  if (Lib.isString(column)) {
+  if (Lib.isString(column) || Lib.isStringLike(column)) {
```

Comment:
@lbrdnk Is there anything preventing us from using the new abstraction here as well?
```suggestion
  if (Lib.isStringOrStringLike(column) {
```